gcore/utils.go

```diff
@@ -44,6 +45,9 @@ const (
 
 	projectPoint = "projects"
 	regionPoint  = "regions"
+
+	ConflictRetryAmount   = 180
```

Comment:
we should always calculate it based on 
```
d.Timeout(schema.TimeoutCreate).Seconds() / ConflictRetryInterval (received from env var or default)
```

Reply:
calculated based on resource timeout and conflict retry interval (either default or provided by user, set to 0 in case user provides 0 as interval)